src/Directory.Build.props

```diff
@@ -171,10 +171,4 @@
 
 	<Import Project="Uno.CrossTargeting.props" />
 
-	<ItemGroup>
-		<InternalsVisibleTo Include="$(AssemblyName).UI" />
```

Comment:
Why have these internalsvisibleto items been removed?

Reply:
Because they are causing lot of troubles with "compiler compatibility classes" like `ModuleInitializerAttribute`: When we reference 2 projects that are defining them, the compiler is not able to choose one.

IMO We should almost never use the "InternalsVisibleTo", even for tests, if we are testing specific internal implementation, I started to prefer reflection over it.